lasy/utils/mode_decomposition.py

```diff
@@ -27,8 +27,11 @@ def hermite_gauss_decomposition(laserProfile, n_x_max=12, n_y_max=12, res=1e-6):
     laserProfile : class instance
         An instance of a class or sub-class of TransverseLaserProfile
 
-    n_x_max, n_y_max : ints
-        The maximum values of `n_x` and `n_y` out to which the expansion
+    wavelength : float (in meter)
+        Central wavelength at which the Hermite-Gauss beams are to be defined.
+
+    m_max, n_max : ints
+        The maximum values of `m` and `n` out to which the expansion
```

Comment:
You know better, just pointing out.
```suggestion
        The maximum values of `m` and `n` up to which the expansion
```